### DISCUSSION TITLE
missing wave diffraction in WW3

### DISCUSSION THREAD

**Discussion by Biao-Zhao** [NOAA-EMC/WW3#1102] [Q&A]
Hi, everyone! I have been using WW3 for ideal experiments recently, but when I checked the WW3 manual, I found that WW3 does not take wave diffraction into account. In comparison, SWAN does consider wave diffraction by a phase-decoupled method. Does anyone know how to account for wave diffraction in WW3? Alternatively, is the WW3 development team planning to introduce diffraction in the near future? Thanks in advance.

**Comment by Biao-Zhao** [comment]
According to my understanding, the diffusion terms in WW3, which are typically used to mitigate the Garden Sprinkler Effect (GSE), cause waves to diffract to some extent behind obstacles. The question is, whether this diffraction is sufficient? I hope someone could help me explain

**Comment by aronroland** [comment]
hi @Biao-Zhao, 

wave diffraction will be soon (approx. 4 weeks) available within the unstructured version of WW3. GSE will also be soon available on unstructured meshes but this is not what I would use for wave diffraction, even if u r right in claiming that due to the diffusion, there will be a flux in the shadowed region of the obstacles but diffraction is far more complicated in terms of getting the right ammount of energy behind the obstacle.

**Reply by Biao-Zhao** [reply]
hi@aronroland,

I am so glad to hear that you are adding diffraction into WW3 and I totally agree what you said about the complexity of wave energy behind the obstacle . Unfortunately, missing wave diffraction in WW3  limits the possibility of further exploration. I was wondering if you have plan to implement wave diffraction on rectilinear or curvilinear grid?    I look forward to seeing your work published in literature, and doing my research using WW3 with more complete processes.

**Comment by aronroland** [comment]
... at this time, I have only interest in unstructured grids ...--Sent from my Android phone with GMX Mail. Please excuse my brevity.On 10/29/23, 11:21 AM Biao Zhao ***@***.***> wrote:

   
  ***@***.***, 
  I am so glad to hear that you are adding diffraction into WW3 and I totally agree what you said about the complexity of wave energy behind the obstacle . Unfortunately, missing wave diffraction in WW3 limits the possibility of further exploration. I was wondering if you have plan to implement wave diffraction on rectilinear or curvilinear grid? I look forward to seeing your work published in literature, and doing my research using WW3 with more complete processes.
